include/swift/Frontend/FrontendInputs.h

```diff
@@ -0,0 +1,127 @@
+//===--- FrontendInputs.h -------------------------------------------------===//
```

Comment:
Missing the `-*- c++ -*-` for the emacs folks.

Reply:
Supplied that for all my new files.